src/Persistence/PaYaml/Models/SchemaV2_2/ComponentDefinition.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.PowerPlatform.PowerApps.Persistence.PaYaml.Models.PowerFx;
+using YamlDotNet.Serialization;
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.PaYaml.Models.SchemaV2_2;
+
+public enum ComponentPropertyKind
+{
+    Input,
```

Comment:
I think there is enum for that already unless you want this in separate namespace

Reply:
this is an enum specifically mapping to the V2.2 schema.
Serialization OM != Runtime OM. ;)

Reply:
won't fix